files/zh-tw/web/javascript/guide/using_promises/index.md

```diff
@@ -192,7 +192,7 @@ async function foo() {
 }
 ```
 
-這基於 Promise，例如 `doSomething()` 和之前一樣。你可以閱讀在[這裡](https://developers.google.com/web/fundamentals/getting-started/primers/async-functions)閱讀更多。
+這基於 Promise，例如 `doSomething()` 和之前一樣。你可以閱讀在[這裡](https://web.dev/)閱讀更多。
```

Comment:
the link was no longer maintained on google.com, and was redircted to web.dev's homepage

Reply:
I'm not agree with the change here. The page `https://web.dev/` does not provide any information about `Promise`.

Reply:
yeap, while the original link neither - also the link was removed in mdn/content

Reply:
```suggestion
這基於 Promise，例如 `doSomething()` 和之前一樣。你可以閱讀在[這裡](http://web.archive.org/web/20170808153026/https://developers.google.com/web/fundamentals/getting-started/primers/async-functions)閱讀更多。
```

Reply:
or else this: https://web.dev/articles/async-functions